examples/image-load.yaml

```diff
@@ -0,0 +1,9 @@
+volume: raw-volume # this needs to match the volume name passed to the podman run command
+targets:
+- name: fetchit
+  url: https://github.com/cooktheryan/fetchit
+  methods:
+    image:
+      targetPath: examples/imageLoad
+      schedule: "*/1 * * * *"
+  branch: imageloading
```

Comment:
is that the branch you want or do you want this branch `image`?

Reply:
good catch